site/learn/expert-advice/identity-basics/open-source-vs-commercial.md

```diff
@@ -0,0 +1,127 @@
+
+This is another thing you don't have to worry about with a commercial provider, rather than being responsible for reviewing all these licenses if you went with an open-source solution.
+
+### Maintenance Responsibility
+
+This is one of the main benefits of a commercial provider. They are responsible for all the maintenance of their product. Because you're paying for the product directly, this is one area you no longer have to worry about. You are trading money for time.
+
+However, unlike with open source, you have less freedom to make fixes yourself. So if you have a feature or a fix that you believe should be in the product, the most you can do is try to convince someone at the vendor to get that prioritized. With an open-source solution, you could fix it yourself, if you are willing to accept the maintenance burden.
+
+If you are worried about access to the source code if the commercial vendor goes out of business, ask about a source code escrow. Some providers are willing to add a provision to their contract with you.
+
+### Security Responsibility
+
+As with maintenance responsibility, the responsibility for the security of a commercial product is on the vendor. They will be the ones applying patches and (hopefully) doing frequent security audits to ensure their product is secure. These could include running internal security reviews, a bug bounty program, or hiring a pentesting firm to test the product's security.
+
+It may take longer to find bugs than it would with an open-source product where more people from all sorts of different backgrounds are invested in the security of the product and review it frequently. While the security responsibility may not be on you as a customer, you _do_ have the responsibility to [perform due diligence on any potential vendor](/learn/expert-advice/identity-basics/due-diligence-authentication-vendors/) and ensure that they take the security of their product seriously. One way to do that is to examine the vendor's public security policy. Another is to look at the [CVE database](https://cve.mitre.org/) and see how vulnerabilities are handled.
+
+### Cost
+
+One of the more notable differences between open source and commercial solutions is cost. Because of the benefits detailed above, there is usually a cost to a commercial solution, whereas many open-source solutions are provided free of charge and maintained by the community. 
+
+However, some commercial software offers free tiers with limited support. For example, Auth0 allows you a certain number of users for free. FusionAuth has a community edition allowing you an unlimited number of users if you self-host.
+
+Depending on what stage your company is in, this cost can either be something you can absorb or a dealbreaker.
```

Comment:
Maybe add something about cost coming with ease of use and features that you wouldn't get with an open source solution

Reply:
We could also elaborate on this in "The Final Showdown" section